### PR TITLE
Changed ${/#ae_reason} to a string value in on_error methods.

### PR DESCRIPTION
**Pull request by billfitzgerald0120**
Changed all occurrences in class schemas and instances.

https://bugzilla.redhat.com/show_bug.cgi?id=1439845

@miq-bot add_label bug, fine/yes euwe/yes

### CODE REVIEW

**Comment by billfitzgerald0120** [comment]
@miq-bot add_label fine/yes, euwe/yes

File: content/automate/ManageIQ/Cloud/Orchestration/Provisioning/StateMachines/Provision.class/__class__.yaml [hunk 1]
```diff
@@ -148,7 +148,7 @@ object:
       condition: 
       on_entry: update_serviceprovision_status(status => 'Creating Stack')
       on_exit: update_serviceprovision_status(status => 'Creating Stack')
-      on_error: update_serviceprovision_status(status => '${/#ae_reason}')
+      on_error: update_serviceprovision_status(status => 'Error Provisioning Stack')
```
Comment (tinaafitz) [inline content/automate/ManageIQ/Cloud/Orchestration/Provisioning/StateMachines/Provision.class/__class__.yaml]: @billfitzgerald0120 Can we change the status to 'Error Creating Stack'?

File: content/automate/ManageIQ/Cloud/Orchestration/Provisioning/StateMachines/Provision.class/__class__.yaml [hunk 2]
```diff
@@ -168,7 +168,7 @@ object:
       condition: 
       on_entry: 
       on_exit: update_serviceprovision_status(status => 'Creating Stack')
-      on_error: update_serviceprovision_status(status => '${/#ae_reason}')
+      on_error: update_serviceprovision_status(status => 'Error Provisioning Stack')
```
Comment (tinaafitz) [inline content/automate/ManageIQ/Cloud/Orchestration/Provisioning/StateMachines/Provision.class/__class__.yaml]: @billfitzgerald0120 Can we change the status to 'Error Creating Stack'?

File: content/automate/ManageIQ/Cloud/VM/Provisioning/StateMachines/VMProvision_VM.class/__class__.yaml [hunk 3]
```diff
@@ -208,7 +208,7 @@ object:
       condition: 
       on_entry: 
       on_exit: update_provision_status(status => 'Creating VM')
-      on_error: update_provision_status(status => '${/#ae_reason}')
+      on_error: update_provision_status(status => 'Error Provisioning Instance')
```
Comment (tinaafitz) [inline content/automate/ManageIQ/Cloud/VM/Provisioning/StateMachines/VMProvision_VM.class/__class__.yaml]: @billfitzgerald0120 Can we change the status to 'Error Creating VM'?

**Comment by billfitzgerald0120** [comment]
Made changes as requested.
@tinaafitz 
Please Review

**Comment by tinaafitz** [comment]
@billfitzgerald0120 Looks good.
@mkanoor Please review.

**Comment by simaishi** [comment]
Euwe backport (to manageiq repo) details:

```
$ git log -1
commit 48ad6261b1300039dbf96a6b23e40ccda3e890ea
Author: Madhu Kanoor <mkanoor@redhat.com>
Date:   Thu May 4 14:33:03 2017 -0400

    Merge pull request #98 from billfitzgerald0120/ae_reason_to_string
    
    Changed ${/#ae_reason} to a string value in on_error methods.
    (cherry picked from commit b92d4a4c51704200ce0f143912c4c3866c69b796)
    
    https://bugzilla.redhat.com/show_bug.cgi?id=1449407

**Comment by simaishi** [comment]
Fine backport details:

```
$ git log -1
commit 48721aa4bb3fef7dfadc35408cde96c24b7319e0
Author: Madhu Kanoor <mkanoor@redhat.com>
Date:   Thu May 4 14:33:03 2017 -0400

    Merge pull request #98 from billfitzgerald0120/ae_reason_to_string
    
    Changed ${/#ae_reason} to a string value in on_error methods.
    (cherry picked from commit b92d4a4c51704200ce0f143912c4c3866c69b796)
    
    https://bugzilla.redhat.com/show_bug.cgi?id=1459260
```

**Pull request merged**